.gitlab/includes/clang11_pipeline.yml

```diff
@@ -69,6 +88,7 @@ clang11_debug_test:
   needs:
     - clang11_debug_build
   script:
+    - spack env activate .
```

Comment:
Move this down to the `build-env` command: `spack -e pika... build-env $spack_spec ...`?